lib/auth_app_web/router.ex

```diff
@@ -54,32 +54,33 @@ defmodule AuthAppWeb.Router do
       on_mount: [{AuthAppWeb.UserAuth, :redirect_if_user_is_authenticated}] do
       live "/users/register", UserLive.Registration, :new
       live "/users/log-in", UserLive.Login, :new
-      live "/users/reset-password", UserLive.ForgotPassword, :new
-      live "/users/reset-password/:token", UserLive.ResetPassword, :edit
+      live "/users/log-in/:token", UserLive.Confirmation, :new
+      post "/users/log-in", UserSessionController, :create
     end
-
-    post "/users/log-in", UserSessionController, :create
   end
 
   scope "/", AuthAppWeb do
     pipe_through [:browser, :require_authenticated_user]
```

Comment:
Maybe it is a good opportunity to go with simpler names? `:authenticate_user`?

Reply:
I think require_authenticated_user is clearer, actually